apps/ws/src/index.ts

```diff
@@ -11,8 +10,8 @@ const gameManager = new GameManager();
 wss.on('connection', function connection(ws, req) {
   //@ts-ignore
   const token: string = url.parse(req.url, true).query.token;
-  const userId = extractUserId(token);
-  gameManager.addUser(new User(ws, userId));
+  const User = extractAuthUser(token, ws);
```

Comment:
lets stick with lower case first letter for vars

Reply:
changed to camel case